scapy/arch/__init__.py

```diff
@@ -41,8 +41,12 @@ def str2mac(s):
 
 
 def get_if_addr(iff):
```

Comment:
Alessio, could you add the following docstring ?
"""Return the packed IPv4 address configured on the interface 'iff'."""

Thanks.